cmd/swarmctl/service/flagparser/flags.go

```diff
@@ -51,6 +51,8 @@ func AddServiceFlags(flags *pflag.FlagSet) {
 
 	flags.String("log-driver", "", "specify a log driver")
 	flags.StringSlice("log-opt", nil, "log driver options, as key value pairs")
+
+	flags.Int64("pids-limit", -1, "set container pid limit")
```

Comment:
What will happen if the field value is 0, as would be the case if the service was created by an older version of the code that didn't know about the field?